crates/ty_python_semantic/src/suppression.rs

```diff
@@ -184,6 +186,8 @@ fn check_invalid_suppression(context: &mut CheckSuppressionsContext) {
 ///
 /// Does nothing if the [`UNUSED_IGNORE_COMMENT`] rule is disabled.
 fn check_unused_suppressions(context: &mut CheckSuppressionsContext) {
+    use std::fmt::Write as _;
```

Comment:
nit: is there a reason to import this locally rather than globally?

Reply:
Mainly laziness. RustRover can't auto import `Write` and having to scroll to the top and down again breaks my entire flow.